node-scripts/src/pipeline3/start-instance.js

```diff
@@ -17,7 +17,6 @@ const db = require(cst.DBPATH);
 
 const scripts = require("./scripts");
 
-const secrets = config.getSecrets();
```

Comment:
Please keep this line. Even though it is not used, I like that it is documented how it can be leveraged on.
you can comment it though:
```
// Fetch secrets:
// const secrets = config.getSecrets();
```